src/tests/test_send_notifications.py

```diff
@@ -0,0 +1,46 @@
+"""Test Suite for all Notifications."""
+import os
+
+from .base_test import BaseTestCase
+from api.utils.notifications.email_notices import send_email
```

Comment:
Mock this function, making network call during testing is not a good idea, it adds an extra dependency(internet connection) on our tests which is not important to us, all we need is the behavior of our app when the email is successful or when it's not: whether send email sends the email or not is a concern for the people who designed that package. @kimani-ndegwa any thoughts?

Reply:
That definitely does make sense. If indeed the purpose is to test our code, then you should mock the email functionality.